raiden/tests/integration/api/rest/test_rest.py

```diff
@@ -669,60 +669,80 @@ def test_token_events_errors_for_unregistered_token(api_server_test_instance):
 @raise_on_failure
 @pytest.mark.parametrize("number_of_nodes", [3])
 @pytest.mark.parametrize("enable_rest_api", [True])
+@pytest.mark.parametrize("number_of_tokens", [2])
 def test_payment_events_endpoints(
     api_server_test_instance: APIServer, raiden_network, token_addresses
 ):
     app0, app1, app2 = raiden_network
-    amount1 = PaymentAmount(10)
-    identifier1 = PaymentID(42)
-    secret1, secrethash1 = factories.make_secret_with_hash()
+
     token_address = token_addresses[0]
+    token_address2 = token_addresses[1]
```

Comment:
it would be better to use `token_address0` and `token_address1`, the number at the end is the position in the list.

Reply:
done